### PR TITLE
CAF-1948: Introducing 5 readme files with brief descriptions.

### PR DESCRIPTION
**Pull request by zaramckeown**
Jira: https://jira.autonomy.com/browse/CAF-1948

### CODE REVIEW

File: cipher-null/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# cipher-null
+
+---
+
+Provides a mechanism to encrypt and decrypt a string in Java.
```
Comment (dermot-hardy) [inline cipher-null/README.md]: Is this definitely correct?  The use of the word null suggests to me that this implementation doesn't actually do encryption.

File: codec-json-lzf/README.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# codec-json-lzf 
+
+---
+
+Implementation of Codec that supports Serialization and deserialization to and from JSON. It is compressed with a high-speed LZF algorithm. In some crude tests, this resulted in data that was approximately 60% of the original JSON size with negligible performance impact.
```
Comment (dermot-hardy) [inline codec-json-lzf/README.md]: Don't use past tense for this.

File: codec-yaml/README.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# codec-yaml
+
+---
+
+Utilizes SnakeYAML to serialize/deserialize data. Within it there are two different versions:
```
Comment (dermot-hardy) [inline codec-yaml/README.md]: Drop "Within it"

**Comment by glucy** [comment]
Look fine to me.

**Pull request merged**